mission_control/navigator_launch/nodes/network_keepalive.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+
+
+class KeepAlive(object):
+    '''
+    Meant to run from any number of remote computers.
+    Make sure the wifi link is somewhere between the computers running the keepalive
+        script and the boat (or else there is no point!)
+    '''
+    def __init__(self):
+        self.pub = rospy.Publisher('/keep_alive', String, queue_size=1)
+        self.timer = rospy.Timer(rospy.Duration(0.1), self.keepalive_pub)
+
+    def keepalive_pub(self, *args):
+        self.pub.publish(String('keep_alive'))
+
+
+if __name__ == '__main__':
+    rospy.init_node('network_keepalive')
+    KeepAlive()
```

Comment:
Really? I'm surprised, I was trying to find out how often the python garbage collector looked for unreferenced objects but didnt find the answer.

Yes, that would be much better. All hail the dark one.